services/webdav/pkg/config/config.go

```diff
@@ -20,6 +20,7 @@ type Config struct {
 
 	HTTP HTTP `yaml:"http"`
 
+	DisablePreviews      bool            `yaml:"disablePreviews" env:"OCIS_DISABLE_PREVIEWS;WEBDAV_DISABLE_PREVIEWS" desc:"Set this option to 'true' to disable all preview related webdav paths."`
```

Comment:
Reading this many times, it may be clear to developers but it is very hard to understand for admins. Maybe one has a better idea for the wording.